database/clickhouse/clickhouse.go

```diff
@@ -21,6 +22,7 @@ var (
 	multiStmtDelimiter = []byte(";")
 
 	DefaultMigrationsTable       = "schema_migrations"
+	DefaultTableEngine           = "TinyLog"
```

Comment:
Same, rename to `DefaultMigrationsTableEngine`

Reply:
same